website/docs/r/mssql_server.html.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 subcategory: "Database"
 layout: "azurerm"
-page_title: "Azure Resource Manager: azurerm_mssql_server"
+page_title: "Azure Resource Manager: azurerm_sql_server"
```

Comment:
I don't think we should be making this change?

Reply:
Oops, not sure how I did that!